content/research_report/introductions.md

```diff
@@ -1,61 +1,70 @@
 For example, when functional connectivity is contaminated with motion, it can lead to altered network structure {cite:p}`power_scrubbing_2012`.
 Indeed, in younger subjects and neurologic/psychiatric patients the impact of head motion on connectivity is frequently reported {cite:p}`makowski_head_2019,satterthwaite_impact_2012`.
 Hence, reducing the impact of confounds, known as denoising,
 is a major component of the fMRI workflow to ensure the quality of analysis.
 
-<!-- Classes of nuisance regressors - like how load_confounds separate them -->
-<!-- need to add reference to this section-->
 ## Types of noise regressors
 
-<!-- NC: I thought the opening and closing sentences of this section were repeating the same informtion unecessarily -->
-<!-- so I combined them. It could also work at the end, but I think this way flows better to the next section -->
+<!-- aim of the paragraph: Classes of nuisance regressors - like how load_confounds separate them -->
 The most common method to minimize the impact of confounds in functional connectivity analysis is to perform a linear regression {cite:p}`friston_statistical_1994`.
 After basic processing steps (see fMRIPrep {cite:p}`fmriprep1`), 
 regressors are regressed out from the signal and the resulting residual used as denoised signal in all subsequent analyses.
 Nuisance regressors can be separated into classes, capturing different types of non-neuronal noise.
-__Temporal high-pass filtering__ accounts for low-frequency signal drifts introduced by _physiological and scanner noise sources_.
-__Head motion__ is captured by _6 rigid-body motion parameters_ (3 translations and 3 rotation) estimated relative to a reference image {cite:p}`friston_movement-related_1996`.
+__Head motion__ is captured by motion realignment measures: _6 rigid-body motion parameters_ (3 translations and 3 rotation) estimated relative to a reference image {cite:p}`friston_movement-related_1996`.
 __Non-grey matter tissue signal__ (such as _white matter_ and _cerebrospinal fluid_), unlikely to reflect neuronal activity {cite:p}`fox_pnas_2005`,
 is captured by averaging signal within anatomically-derived masks.
 __Global signal__ is calculated by averaging signal within the _full brain_ mask {cite:p}`fox_pnas_2005`.
+These three classes of regressors can be expanded to their first temporal derivatives and their quadratic terms {cite:p}`satterthwaite_2013`.
+__Scrubbing__ {cite:p}`power_scrubbing_2012` is a volume censoring approach to remove high motion segments which the framewise displacement exceeds some threshold. 
+The scrubbing approach is applied alogn side head motion parameters and tissue signal regressors.
+__Temporal high-pass filtering__ accounts for low-frequency signal drifts introduced by _physiological and scanner noise sources_.
+Aside from regressors directly modeling regressors derived from realignment measures or anatomical properties,
```

Comment:
I think 'regressors directly modeling regressors' doesn't really make sense. Could you use another word? Modeling noise perhaps?

Reply:
yes modeling noise definitely makes more sense